src/Behat/Context/EntityContext.php

```diff
@@ -240,7 +240,9 @@ protected function replaceTokens($values) {
         if (!empty($entity)) {
           $values[$key] = $this->getCore()->getEntityFieldValue($destiny_replacement, $entity, $values[$key]);
         }
-
+      }
+      elseif (strpos($value, 'relative-date') === 0) {
+        $values[$key] = strtotime($value);
```

Comment:
Reading this it looks like $value contains 'relative-date' inside. `strtotime` won't return a correct timestamp. Please correct it.